content/rc/how-to/flush-db.md

```diff
@@ -35,4 +35,12 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
```

Comment:
1. I didn't understand what the following sentence means: "When you flush a CRDB, multiple resources flush their data simultaneously". Maybe it worth saying that flush a CRDB will remove / erase all the data over all replicas with no way to recover it (unless you have a backup)
2. Please add also the possibility to flush the CRDB using the REST API (as written in ticket- RED-25549 by Yossi)

Reply:
2. I added the API instructions below

Reply:
1. Sounds good to me.